### PR TITLE
updated pack shot url to 2017 version

### PR DESCRIPTION
**Pull request by jkerr321**
@quarterto updated hardcoded packshot to latest version as discussed :)

### CODE REVIEW

File: views/partials/barrier.html [hunk 1]
```diff
@@ -28,7 +28,7 @@ <h2 class="amp-access-barrier__offer-title">
...
 			\{{/productImage}}
 			\{{^productImage}}
-			{{> link href='{{#href}}{{href}}{{/href}}{{^href}}https://www.ft.com/signup?offerId={{id}}{{/href}}' linkText='{{title}}' text='<amp-img class="amp-access-barrier__offer_image" layout="responsive" width="300" height="150" src="https://www.ft.com/__origami/service/image/v2/images/raw/https:%2F%2Fwww.ft.com%2F__assets%2Fcreatives%2Fstandard_digital.png?width=800&source=next&fit=scale-down"></amp-img>' type='barrier-image'}}
+			{{> link href='{{#href}}{{href}}{{/href}}{{^href}}https://www.ft.com/signup?offerId={{id}}{{/href}}' linkText='{{title}}' text='<amp-img class="amp-access-barrier__offer_image" layout="responsive" width="300" height="150" src="https://www.ft.com/__origami/service/image/v2/images/raw/https%3A%2F%2Fft.com%2F__assets%2Fcreatives%2Fproduct%2Fmarch2017-pack-shots%2FMar2017_StandardDigital.png?source=FT&width=800&fit=scale-down"></amp-img>' type='barrier-image'}}
```
Comment (quarterto) [inline views/partials/barrier.html]: I know it was `next` before but this should be `source=amp` 😬

Comment (jkerr321) [inline views/partials/barrier.html]: ahhh interesting! No worries, will change it

Comment (jkerr321) [inline views/partials/barrier.html]: Done @quarterto 😄

**Comment by quarterto** [comment]
eep, entirely forgot about this, merging

**Pull request merged**